lib/util/problem.js

```diff
@@ -140,6 +140,8 @@ const problems = {
 
     encodingNotSupported: problem(400.37, () => 'Encoding not supported.'),
 
+    passwordTooLong: problem(400.38, () => 'The password or passphrase provided exceeds the maximum length.'),
```

Comment:
Are you thinking this is likely rare enough that we just leave a raw backend message (vs. introducing localizable frontend error messages)?

Reply:
What's the distinction?  Should there be an accompanying PR for `odk-central-frontend`?